Sources/CALayer+ContentsGravity.swift

```diff
@@ -75,6 +72,9 @@ struct ContentsGravityTransformation {
         case .topRight:
             offset = CGPoint(x: distanceToMaxX, y: distanceToMinY)
             scale = .defaultScale
+        default:
```

Comment:
we should have no default case here. this will confuse people. I noticed `.bottomLeft` and `.bottomRight` are new: would be great if you could either comment them out for now or implement them here.

Reply:
oh yeah, kind of stupid of me, just wanted a compiling version when I did this ^^